x-pack/legacy/plugins/maps/public/actions/map_actions.js

```diff
@@ -431,7 +470,8 @@ export function updateSourceDataRequest(layerId, newData) {
 }
 
 export function endDataLoad(layerId, dataId, requestToken, data, meta) {
-  return async (dispatch) => {
+  return async (dispatch, getState) => {
```

Comment:
There's nothing async happening here, might want to remove it since you're in here

Reply:
added #37934 (comment) to issue tracking clean-up of actions. Want to avoid making changes in this PR that are outside scope